content/id/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Poseidon-Firmament - Sebuah Penjadwal Alternatif
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+**Rilis saat ini dari Penjadwal Poseidon-Firmament adalah rilis <code> alfa </code>.**
+
+Penjadwal Poseidon-Firmament adalah penjadwal alternatif yang dapat digunakan bersama penjadwal Kubernetes bawaan.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+
+## Pengenalan	
+
+Poseidon adalah sebuah layanan yang berperan sebagai pemersatu antara [Penjadwal Firmament](https://github.com/Huawei-PaaS/firmament) dengan Kubernetes. Penjadwal Poseidon-Firmament menambah kapabilitas penjadwal Kubernetes saat ini. Penjadwal ini menggabungkan kemampuan penjadwalan berbasis aliran jaringan grafis baru bersama penjadwal Kubernetes bawaan. Penjadwal Firmament memodelkan beban kerja dan kelompok sebagai jaringan aliran dan menjalankan optimisasi aliran biaya-minimum kepada jaringan ini untuk membuat keputusan penjadwalan.
+
+Penjadwal ini memodelkan masalah penjadwalan sebagai optimasi berbasis kendala atas grafik jaringan aliran. Hal ini dicapai dengan mengurangi penjadwalan ke masalah optimisasi biaya-minimum aliran-maks. Penjadwal Poseidon-Firmament secara dinamis memperbaiki penempatan beban kerja.
+
+Penjadwal Poseidon-Firmament berjalan bersamaan dengan penjadwal Kubernetes default sebagai penjadwal alternatif, sehingga beberapa penjadwal dapat berjalan secara bersamaan.
+
+## Keuntungan Utama
+
+### Penjadwalan grafik berbasis penjadwalan Poseidon-Firmament memberikan beberapa keuntungan utama sebagai berikut:
+
+- Beban kerja (_pod_) dijadwalkan secara kolektif untuk memungkinkan penjadwalan dalam skala besar.
+- Berdasarkan hasil tes kinerja yang luas, skala Poseidon-Firmament jauh lebih baik daripada penjadwal bawaan Kubernetes dilihat dari jumlah node meningkat dalam sebuah cluster. Hal ini disebabkan oleh fakta bahwa Poseidon-Firmament mampu mengamortisasi lebih banyak pekerjaan di seluruh beban kerja.
+- Penjadwal Poseidon-Firmament mengungguli penjadwal bawaan Kubernetes dengan margin lebar ketika menyangkut jumlah kinerja _throughput_ untuk skenario di mana kebutuhan sumber daya komputasi agak seragam di seluruh pekerjaan (_Replicaset_ / _Deployment_ / _Jobs_). Angka kinerja _throughput_ _end-to-end_ penjadwal Poseidon-Firmament , termasuk waktu _bind_, secara konsisten menjadi lebih baik seiring jumlah node dalam sebuah cluster meningkat. Misalnya, untuk klaster 2.700 _node_ (ditampilkan dalam grafik [di sini] (https://github.com/kubernetes-sigs/poseidon/blob/master/docs/bADmark/README.md)), penjadwal Poseidon-Firmament berhasil mencapai 7X atau _throughput_ _end-to-end_ yang lebih besar daripada penjadwal bawaan Kubernetes, yang mencakup waktu _bind_.
```

Comment:
singular is preferred for english word, and use camelCase for API resource name
```suggestion
- Penjadwal Poseidon-Firmament mengungguli penjadwal bawaan Kubernetes dengan margin lebar ketika menyangkut jumlah kinerja _throughput_ untuk skenario di mana kebutuhan sumber daya komputasi agak seragam di seluruh pekerjaan (Replicaset / Deployment / Job). Angka kinerja _throughput_ _end-to-end_ penjadwal Poseidon-Firmament , termasuk waktu _bind_, secara konsisten menjadi lebih baik seiring jumlah Node dalam sebuah kluster meningkat. Misalnya, untuk kluster 2.700 Node (ditampilkan dalam grafik [di sini] (https://github.com/kubernetes-sigs/poseidon/blob/master/docs/bADmark/README.md)), penjadwal Poseidon-Firmament berhasil mencapai 7X atau _throughput_ _end-to-end_ yang lebih besar daripada penjadwal bawaan Kubernetes, yang mencakup waktu _bind_.
```

Reply:
done